cmd/cmd_test.go

```diff
@@ -18,6 +18,8 @@ const cfgHomeKey = "EXERCISM_CONFIG_HOME"
 // The args are the faked out os.Args. The first two arguments
 // in the Args will be ignored. These represent the command (e.g. exercism)
 // and the subcommand (e.g. download).
+// Pass any interactive responses needed for the test in a single
+// String in MockInput, separated by newlines.
```

Comment:
This is great, thank you!

I agree that being able to test this properly is very useful indeed. No need to add it in a separate PR, it makes sense to keep it here.